dev/build/windows/makecoq_mingw.sh

```diff
@@ -1971,6 +1971,37 @@ function make_addon_gappa {
   fi
 }
 
+# HoTT Homotopy Type Theory
+
+function make_addon_hott {
+  if build_prep_overlay HoTT ; then
+    installer_addon_section hott "HoTT" "Homotopy Type Theory" ""
+    logn autogen ./autogen.sh
```

Comment:
You should maybe use -skip-submodules to avoid wasting time/bandwidth cloning coq

Reply:
In which step are the sub modules cloned? Does autogen or configure do this? `build_prep_overlay HoTT` just downloads a `.tar.gz` file from the  git repo server.